### ISSUE TITLE
Pre Launch Task error message not interpreted correctly

### ISSUE THREAD

**Issue by rebornix** [Microsoft/vscode#23480]
#23277 

If users write pre task incorrectly, the error message is as below

```
There is a task {0} running. Can not run pre launch task watch.
```

**Comment by rebornix** [comment]
This happens in Terminal Runner. Might be a dup. Didn't notice we are only testing against the old runner.

**Comment by dbaeumer** [comment]
@isidorn moving to you since you output that message. Please ping if there is something you need from the task side

**Comment by isidorn** [comment]
@dbaeumer you need to make sure that every task event has a name, otherwise this can not happen
[Code pointer](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/debug/electron-browser/debugService.ts#L129)

What does it mean if I get a task event without a name?

**Comment by dbaeumer** [comment]
@isidorn according to @rebornix this is on the new terminal runner so I will move to April

@rebornix I tried to reproduce this but wasn't able to

**Comment by rebornix** [comment]
@dbaeumer I forgot to put the reproduce steps, here it is

* Open VS Code repository. Add `"_runner": "terminal"` to our `.vscode/task.json`
* Add `"preLaunchTask": "watch"` to `Launch VS Code` session.
* F5

**Comment by dbaeumer** [comment]
This got fixed for the recovery build 1.11.2
